osh/builtin_process.py

```diff
@@ -258,12 +258,7 @@ def Run(self, cmd_val):
     # TODO: Print job ID rather than the PID
     log('Continue PID %d', pid)
     posix.kill(pid, signal.SIGCONT)
-
-    job = self.job_state.JobFromPid(pid)
-    job.state = job_state_e.Running  # needed for Wait() loop to work
-    status = job.Wait(self.waiter)
-    #log('status = %d', status)
-    return status
+    return self.job_state.NotifyContinue(pid, self.waiter)
```

Comment:
looks like this return statement isn't necessary

Reply:
I did this to preserve the existing behavior, but I'll take a closer look.

Reply:
I looked through the other classes in `builtin_process.py`, and it seems like they all their `Run` methods return some kind of status code. Am I missing something?